packages/storage/test/integration/integration.test.ts

```diff
@@ -209,7 +209,7 @@ describe('FirebaseStorage Exp', () => {
     );
     await Promise.race([
       failureDeferred.promise,
-      new Promise(resolve => setTimeout(resolve, 4000))
+      new Promise(resolve => setTimeout(resolve, 2000))
```

Comment:
I'm a little nervous about this one - I'm guessing it was set to 4000 to give enough time for an error to happen if it was going to happen. It's possible that a task that might have been on track to error hadn't gotten to the point where it's calling the error callback at 2000ms? Is the problem that the mocha/karma test timeout is expiring before this test case finishes? In that case can we extend the test timeout for this specific test? I think there's a way to do it.

If this timeout problem only exists in the Node test then I think that this is the syntax for mocha. https://mochajs.org/#test-level I don't know if it causes any syntax problems when you run the test in karma (the browser tests), if so you can do a conditional to test that it exists or is settable first?

Reply:
You're right, I think just increasing the timeout at the test level is a better idea- I didn't know you could do that!

The test passes after 5249ms after this https://github.com/firebase/firebase-js-sdk/pull/8443/commits/f65709ec961f37487f2cc240583e28cb3e95ca37